### PR TITLE
Updated CHANGELOG.md

### PR DESCRIPTION
**Pull request by Big-Zude**

### CODE REVIEW

**Comment by OlivierJM** [comment]
#74

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,16 @@
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased version]
+
+### v1.7.8.0-alpha (February 6, 2019)
+- This release contains bugs, that will be fixed in the final release, use it only for testing.
+
+- What is new?
```
Comment (OlivierJM) [inline CHANGELOG.md]: Remove the "what is new ?" line

Comment (Big-Zude) [reply CHANGELOG.md]: i removed it

Comment (OlivierJM) [reply CHANGELOG.md]: cool. Thanks

**Comment by OlivierJM** [comment]
This is awesome @Big-Zude

**Pull request merged**